### PR TITLE
Clean code

### PR DESCRIPTION
**Pull request by PavelYudzin**

### CODE REVIEW

File: clean-code/Java/src/main/java/Airport.java [hunk 1]
```diff
@@ -1,110 +1,104 @@
...
+        }
+        return transportMilitaryPlanes;
     }
 
     public List<MilitaryPlane> getBomberMilitaryPlanes() {
         List<MilitaryPlane> bomberMilitaryPlanes = new ArrayList<>();
         List<MilitaryPlane> militaryPlanes = getMilitaryPlanes();
-        for (int i = 0; i < militaryPlanes.size(); i++) {
-            MilitaryPlane plane = militaryPlanes.get(i);
+        for (MilitaryPlane plane : militaryPlanes) {
```
Comment (VitaliLebedzeu) [inline clean-code/Java/src/main/java/Airport.java]: separate 'for' cycle to another method

File: clean-code/Java/src/main/java/Planes/MilitaryPlane.java [hunk 2]
```diff
@@ -4,10 +4,8 @@
 
 import java.util.Objects;
 
-public class MilitaryPlane extends Plane{
-
-    private MilitaryType type;
-
+public class MilitaryPlane extends Plane {
+    private final MilitaryType type;
     public MilitaryPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity, MilitaryType type) {
```
Comment (VitaliLebedzeu) [inline clean-code/Java/src/main/java/Planes/MilitaryPlane.java:10]: add the separate line

File: clean-code/Java/src/main/java/Planes/SportPlane.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package Planes;
+
+public class SportPlane extends Plane {
+    public SportPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity) {
+        super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
+
```
Comment (VitaliLebedzeu) [inline clean-code/Java/src/main/java/Planes/SportPlane.java]: remove empty line

File: clean-code/Java/src/main/java/Planes/SportPlane.java [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package Planes;
+
+public class SportPlane extends Plane {
+    public SportPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity) {
```
Comment (VitaliLebedzeu) [inline clean-code/Java/src/main/java/Planes/SportPlane.java:5]: add  separate line

**Review by VitaliLebedzeu — APPROVED** [review]

File: clean-code/Java/src/main/java/Airport.java (hunk 1 above)
Comment (PavelYudzin) [reply clean-code/Java/src/main/java/Airport.java]: Fixed

File: clean-code/Java/src/main/java/Planes/MilitaryPlane.java (hunk 2 above)
Comment (PavelYudzin) [reply clean-code/Java/src/main/java/Planes/MilitaryPlane.java:10]: Fixed

File: clean-code/Java/src/main/java/Planes/SportPlane.java (hunk 3 above)
Comment (PavelYudzin) [reply clean-code/Java/src/main/java/Planes/SportPlane.java]: Fixed

File: clean-code/Java/src/main/java/Planes/SportPlane.java (hunk 4 above)
Comment (PavelYudzin) [reply clean-code/Java/src/main/java/Planes/SportPlane.java:5]: Fixed

**Pull request merged**